changelog/2155.doc.rst

```diff
@@ -0,0 +1,3 @@
+Updated the code contribution workflow instructions in the |contributor guide|
+to reflect that first-time contributors should add themselves to the author
+list in |CITATION.cff|_ instead of in |docs/about/credits.rst|_.
```

Comment:
Usually we haven't put information on GitHub Action updates in changelog entries since they don't affect end users, but we do include updates to the documentation.